python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_auto_materialize_asset_evaluations.py

```diff
@@ -11,7 +11,23 @@
     ExecutingGraphQLContextTestMatrix,
 )
 
-QUERY = """
+SINGLE_QUERY = """
+query GetEvaluationsQuery($assetKey: AssetKeyInput!, $evaluationId: Int!) {
```

Comment:
this has the same name as the multi query

Reply:
fixed